service/github/githubutils/comment_writer.go

```diff
@@ -37,6 +37,8 @@ func (lw *GitHubActionLogWriter) Post(_ context.Context, c *reviewdog.Comment) e
 	return nil
 }
 
+func (*GitHubActionLogWriter) ShouldPrependGitRelDir() bool { return true }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method GitHubActionLogWriter.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChA3YTZjNTgzYTJhYjAzYWY5Eg9nb2xpbnQtY2lyY2xlY2k= -->